docs/quickstart.rst

```diff
@@ -150,8 +150,8 @@ Choosing backend
 Let's consider using external cache storage instead of :class:`lru`.
 
 **Ring** includes common cache storage supports. `Memcached` is one of the
-popular cache storage. `Memcached` itself is out of the Python world. You must
-install and run it to let your python code connects there. Fortunately,
+popular cache storage. `Memcached` itself is outside of the Python world. You
```

Comment:
+1

This is more correct as it states (correctly) that Memcached is not a Python project.

Reply:
Thanks, I made a fix https://github.com/youknowone/ring/commit/97922bd260acd5ac8aad945f95c8de30e1adee71